.github/workflows/anaconda_alpine_ci.yml

```diff
@@ -16,18 +18,50 @@ jobs:
     steps:
       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
 
+      - name: Login to DockerHub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags')
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to Public ECR
```

Comment:
Does public.ecr.aws/y0o4y9o3/anaconda3 exist? Afaik we only push the anaconda-pkg-build images to ECR to have rate limits during package builds.

Reply:
It seemed to [exist](https://gallery.ecr.aws/y0o4y9o3/anaconda3) but did not have anything pushed to it. I will remove the ECR images.